demo/index.html

```diff
@@ -27,18 +27,27 @@
   <link rel="import" href="../paper-icon-item.html">
   <link rel="import" href="../paper-item.html">
   <link rel="import" href="../paper-item-body.html">
+  <link rel="import" href="../../paper-styles/paper-styles.html">
+  <link rel="import" href="../../paper-styles/demo-pages.html">
 
-  <link rel="stylesheet" href="../../paper-styles/demo.css">
-
-  <style>
+  <style is="custom-style">
     .list {
+      padding-top: 12px;
+      background-color: white;
       display: inline-block;
-      border: 1px solid #ccc;
-      padding: 8px 0;
+      width: 240px;
+      height: 228px;
+      margin: 12px;
+      @apply(--shadow-elevation-2dp);
+    }
+
+    .short {
+      padding-top: 12px;
+      height: 216px;
     }
 
-    .wide {
-      min-width: 400px;
+    h4 {
```

Comment:
Are any of these styles useful in other demos? If so, please move to `paper-styles`.